cfme/ansible_tower/explorer.py

```diff
@@ -0,0 +1,144 @@
+    reload = Button(title=VersionPick({Version.lowest(): 'Reload current display',
+                                       '5.9': 'Refresh this page'}))
+    configuration = Dropdown('Configuration')
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class TowerExplorerSystemJobTemplatesToolbar(View):
+    reload = Button(title=VersionPick({Version.lowest(): 'Reload current display',
+                                       '5.9': 'Refresh this page'}))
+    policy = Dropdown('Policy')
+    download = Dropdown('Download')
+    view_selector = View.nested(ItemsToolBarViewSelector)
+
+
+class TowerExplorerView(BaseLoggedInPage):
+    search = View.nested(Search)
+    sidebar = View.nested(TowerExplorerAccordion)
+
+    @property
+    def in_tower_explorer(self):
+        return (self.logged_in_as_current_user and
+                self.navigation.currently_selected == ['Automation', 'Ansible Tower', 'Explorer'])
+
+
+class TowerExplorerProviderView(TowerExplorerView):
+    toolbar = View.nested(TowerExplorerProviderToolbar)
+
+
+class TowerExplorerSystemView(TowerExplorerView):
+    toolbar = View.nested(TowerExplorerSystemJobTemplatesToolbar)
+
+
+class TowerExplorerJobTemplatesView(TowerExplorerView):
+    toolbar = View.nested(TowerExplorerSystemJobTemplatesToolbar)
+
+
+class TowerExplorerProviderDefaultView(TowerExplorerProviderView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_tower_explorer and
+            self.title.text == 'All Ansible Tower Providers' and
+            self.sidebar.tower_explorer_providers.is_opened
+        )
+
+
+class TowerExplorerSystemDefaultView(TowerExplorerProviderView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_tower_explorer and
+            self.title.text == 'All Ansible Tower Configured Systems' and
+            self.sidebar.tower_explorer_systems.is_opened
+        )
+
+
+class TowerExplorerJobTemplatesDefaultView(TowerExplorerProviderView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_tower_explorer and
+            self.title.text == 'All Ansible Tower Job Templates' and
+            self.sidebar.tower_explorer_job_templates.is_opened
+        )
+
+
+class TowerExplorerProvider(Navigatable):
+    pass
+
+
+class TowerExplorerSystem(Navigatable):
+    pass
+
+
+class TowerExplorerJobTemplates(Navigatable):
+    pass
+
+
+@navigator.register(TowerExplorerProvider, 'All')
+class TowerExplorerProviderAll(CFMENavigateStep):
+    VIEW = TowerExplorerProviderDefaultView
+    prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.navigation.select('Automation', 'Ansible Tower', 'Explorer')
```

Comment:
**REQUIRED**
`self.prerequisite_view.navigation.select('Automation', 'Ansible Tower', 'Explorer')` should be a separate destination, let's say `TowerExplorer`:
```python
@navigator.register(Server)
class TowerExplorer(CFMENavigateStep):
    VIEW = TowerExplorerView
    prerequisite = NavigateToSibling('LoggedIn')

    def step(self):
        self.view.navigation.select('Automation', 'Ansible Tower', 'Explorer')
```
Other destinations should use it as a prerequisite.